src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsDynamicFeature.java

```diff
@@ -0,0 +1,53 @@
+package com.wavefront.sdk.jaxrs.server;
+
+import com.wavefront.config.WavefrontReportingConfig;
+import com.wavefront.opentracing.WavefrontTracer;
+import com.wavefront.opentracing.reporting.WavefrontSpanReporter;
+import com.wavefront.sdk.common.WavefrontSender;
+import com.wavefront.sdk.common.application.ApplicationTags;
+import com.wavefront.sdk.jaxrs.reporter.WavefrontJaxrsReporter;
+
+import org.apache.commons.lang3.BooleanUtils;
+
+import javax.ws.rs.container.DynamicFeature;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.Provider;
+
+import io.opentracing.Tracer;
+
+import static com.wavefront.config.ReportingUtils.constructWavefrontSender;
+
+@Provider
+public class WavefrontJaxrsDynamicFeature implements DynamicFeature {
```

Comment:
Javadoc?
Also explain how someone can use WavefrontJaxrsDynamicFeature
i.e. what's the use-case?

Reply:
Basically just adding a few lines of code as below:
```
  @Bean
  public DynamicFeature wavefrontJaxrsFeatureBean(Environment env) {
    ApplicationTags applicationTags =
        constructApplicationTags(env.getProperty("applicationTagsYamlFile"));
    WavefrontReportingConfig wfReportingConfig =
        constructWavefrontReportingConfig(env.getProperty("wfReportingConfigYamlFile"));
    return new WavefrontJaxrsDynamicFeature(applicationTags, wfReportingConfig);
  }
```
Maybe put some example in README.md?